sheetwork/core/sheetwork.py

```diff
@@ -117,6 +125,20 @@ def load_sheet(self):
         logger.debug(f"Loaded SHEET HEAD: {df}")
         self.sheet_df = df
 
+    def count_empty_headers(self, df: pandas.DataFrame) -> int:
+        """Check how many headers are made of whitespaces only.
+
+        Later on, it will raise an issue for the user to provide proper naming in order to write in the database.
```

Comment:
I'm not sure I follow what you this means? Do you mean that it will be used in the code to throw an error?

If so, I'm thinking we could make the error raising directly in this function what do you think?

Then the function could be named something like `assert_no_empty_header_cols()` or something like that. We can even have a arg such as `raise_error: bool = True` which would control whether we `raise` or whether we just send something like a `logger.warning()` message.

Do you buy that?